packages/core/src/NodeView.ts

```diff
@@ -98,7 +98,9 @@ export class NodeView<
       y = handleBox.y - domBox.y + offsetY
     }
 
-    event.dataTransfer?.setDragImage(this.dom, x, y)
+    const clonedNode = this.dom.cloneNode(true) as HTMLElement
```

Comment:
Do we need to clean this up after cloning the node?

Reply:
We don't insert it into the DOM, so I believe we don't need manual cleanup

Reply:
Alright, thanks for the clarification. I'll just double check that myself and will merge when I find it's fine like that. Thanks for your contribution!